### ISSUE TITLE
"jbang alias list" should not show "Bill of Materials"

### ISSUE THREAD

**Issue by koppor** [nbbrd/jbang-catalog#1]
The catalog file contains "Bill of Materials" (see https://github.com/nbbrd/jbang-catalog/blob/2ff3d3db4315266dec3a5951f27d5147faff8662/jbang-catalog.json#L13). This is IMHO wrong.

1. `jbang catalog add https://raw.githubusercontent.com/nbbrd/jbang-catalog/master/jbang-catalog.json`
2. ´jbang alias list`

Output

```
hello-nbbrd = Sandbox - Bill of Materials
heylogs = Keep-a-changelog tool - Bill of Materials
sdmx-dl = Download SDMX data - CLI
```

The third entry is "OKish". However, I would remove the part " - ..." at all entries

```
hello-nbbrd = Sandbox
heylogs = Keep-a-changelog tool
sdmx-dl = Download SDMX data
```

**Comment by charphi** [comment]
This is a bug caused by the use of default values :)

**Comment by charphi** [comment]
Should be fixed with [e8e108becdad23794f517db4a5f051ecccdab08a](https://github.com/nbbrd/heylogs/commit/e8e108becdad23794f517db4a5f051ecccdab08a)
